README.md

```diff
@@ -225,8 +225,19 @@ If only one module is to be added, `modules` is simply the name of the module.
 If multiple images are to be added, `modules` is an array, which holds the names of modules
 to be added, in that particular order.
 
+<<<<<<< HEAD
```

Comment:
Oops missed a rebase conflict! Just this change and we should be good! 👍

Reply:
Sorry for that! Forgot to save the file. Sometimes auto-save is remembered! :smile: 
Thanks